### PR TITLE
[doc] PR for YBA default ports list

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-5011
plus minor fixes in the the yba metrics file

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ Node metrics should be considered on a per-node basis.
 | Consensus Ops / Sec                  | Yugabyte implements the RAFT consensus protocol, with minor modifications.<br><br><br>Update: replicas implement an RPC method called UpdateConsensus which allows a leader to replicate a batch of log entries to the follower. Only a leader may call this RPC method, and a follower can only accept an UpdateConsensus call with a term equal to or higher than its currentTerm.<br>Request: replicas also implement an RPC method called RequestConsensusVote, which is invoked by candidates to gather votes.<br>MultiRaftUpdates: information pending. | A high number for the Request Consensus indicates that a lot of replicas are looking for a new election because they have not received a heartbeat from the leader.<br>A high CPU or a network partition can cause this condition, and therefore it should result in issuing an alert. | ![img](/images/yp/metrics27.png) |
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:103]: Couple of 'TServer' under Total YSQL Connections above

File: docs/content/preview/reference/configuration/default-ports.md [hunk 2]
```diff
@@ -60,31 +60,35 @@ The following common ports are required for firewall rules:
...
 ```
 <target>/prometheus-metrics
 ```
 
-You can access the Prometheus server on port `9090` of the YugabyteDB Anywhere node, and you can see the list of targets at the `http://<yugaware-ip>:9090/targets`. In particular, note port `9300` for node level metrics.
+You can access the Prometheus server on port `9090` of the YugabyteDB Anywhere node, and you can see the list of targets at the `http://<yugaware-ip>:9090/targets`. In particular, note port `9300` for node-level metrics:
+
+| Service            | Port |
+| ------------------ | ---- |
+| Prometheus metrics | 9300 |
```
Comment (iSignal) [inline docs/content/preview/reference/configuration/default-ports.md:73]: `Node metrics` would be a better name for this than `prometheus metrics`. The metrics are ingested into prometheus but node exporter can be used without prometheus as well.

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md [hunk 3]
```diff
@@ -287,11 +287,11 @@ You can also federate metrics from YugabyteDB Anywhere and configure alerting ru
...
+You can check the status of the YB-Master and YB-TServer on each YugabyteDB node by navigating to **Universes > Universe-Name > Nodes**, as per the following illustration:
 
 ![Node Status](/images/yp/troubleshoot-node-status.png)
 
-If issues arise, additional information about each master and TServer is available on their respective **Details** pages, or by accessing `<node_IP>:7000` for master servers and `<node_IP>:9000` for T-Servers (unless the configuration of your on-premises data center or cloud-provider account prevents the access, in which case you may consult [Checking YugabyteDB Servers](../../../troubleshoot/nodes/check-processes/).
+If issues arise, additional information about each master and YB-TServer is available on their respective **Details** pages, or by accessing `<node_IP>:7000` for YB-Master servers and `<node_IP>:9000` for YB-TServers (unless the configuration of your on-premises data center or cloud-provider account prevents the access, in which case you may consult [Checking YugabyteDB Servers](../../../troubleshoot/nodes/check-processes/)).
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:294]: links to page Check YugabyteDB servers

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md [hunk 4]
```diff
@@ -349,13 +349,13 @@ If you are using YugabyteDB Anywhere version 2.12.*n*.*n* and disable **Use Time
 A support bundle is an archive generated at a universe level. It contains all the files required for diagnosing and troubleshooting a problem. The diagnostic information is provided by the following types of files:
 
 - Application logs from YugabyteDB Anywhere.
-- Universe logs, which are the Master and T-Server log files from each node in the universe, as well as Postgres logs  available under the T-Server logs directory.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:352]: PostgreSQL?

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/reference/configuration/default-ports.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/preview/reference/configuration/default-ports.md:73]: Changed

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:103]: fixed

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:294]: fixed

File: docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md (hunk 4 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md:352]: fixed

**Pull request merged**